<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget"/>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QDockWidget" name="Information">
   <attribute name="dockWidgetArea">
    <number>4</number>
   </attribute>
   <widget class="QWidget" name="mainDockWidget">
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QWidget" name="temp" native="true">
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <widget class="QLabel" name="date">
          <property name="minimumSize">
           <size>
            <width>181</width>
            <height>0</height>
           </size>
          </property>
          <property name="text">
           <string>3021年12月12日12时12分星期六</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="blank">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QWidget" name="counts" native="true">
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QLabel" name="points">
          <property name="minimumSize">
           <size>
            <width>91</width>
            <height>0</height>
           </size>
          </property>
          <property name="text">
           <string>点数:9223372036854775807</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="money">
          <property name="minimumSize">
           <size>
            <width>91</width>
            <height>0</height>
           </size>
          </property>
          <property name="text">
           <string>金钱:9223372036854775807</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QWidget" name="Strips" native="true">
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QWidget" name="staminaBarWidget" native="true">
          <layout class="QHBoxLayout" name="horizontalLayout">
           <item>
            <widget class="QLabel" name="staminaBarLabel">
             <property name="minimumSize">
              <size>
               <width>41</width>
               <height>19</height>
              </size>
             </property>
             <property name="text">
              <string>体力:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="MYStrip" name="staminaBarStrip" native="true">
             <property name="minimumSize">
              <size>
               <width>120</width>
               <height>15</height>
              </size>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="staminaBarNumberLabel">
             <property name="minimumSize">
              <size>
               <width>121</width>
               <height>19</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>121</width>
               <height>19</height>
              </size>
             </property>
             <property name="text">
              <string>999999/999999</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QWidget" name="energyBarWidget" native="true">
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <item>
            <widget class="QLabel" name="energyBarLabel">
             <property name="minimumSize">
              <size>
               <width>41</width>
               <height>19</height>
              </size>
             </property>
             <property name="text">
              <string>精力:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="MYStrip" name="energyBarStrip" native="true">
             <property name="minimumSize">
              <size>
               <width>120</width>
               <height>15</height>
              </size>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="energyBarNumberLabel">
             <property name="minimumSize">
              <size>
               <width>121</width>
               <height>19</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>121</width>
               <height>19</height>
              </size>
             </property>
             <property name="text">
              <string>999999/999999</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>MYStrip</class>
   <extends>QWidget</extends>
   <header>Windows/MYStrip.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
